e2e/playwright.config.js

```diff
@@ -0,0 +1,58 @@
+import { defineConfig, devices } from "@playwright/test";
+
+// Load environment variables from .env file if it exists
+// import * as dotenv from 'dotenv';
+// dotenv.config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  timeout: 20000,
+  testDir: "./tests", // Ensure this points to the correct test directory
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: "html",
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: process.env.CI ? undefined : "http://127.0.0.1:3000",
```

Comment:
I don't like the hardcoded localhost URL here, and I also think that the fact that we set the baseURL for CI with `jq` in a script is opaque and hard to track down. I'm thinking we can be more explicit and set this via an environment variable e.g.:

```suggestion
    baseURL: process.env.BASE_URL
```

Reply:
Sounds good - we set it up this way on a11y pr so let's use env in CI and locally. I think I will commit a `.env.example` and gitignore `.env` . I believe `.env` just need `BASE_URL`